ZAPD/Main.cpp

```diff
@@ -258,6 +258,12 @@ void ParseArgs(int& argc, char* argv[])
 	{
 		std::string arg = argv[i];
 
+		// Ignore warning args as they have already been parsed
+		if (arg[0] == '-' && arg[1] == 'W' && arg[2] != '\0')
```

Comment:
Am I correct in understanding that this is meant to apply to arguments which start with `-W` but not `-W` itself?

Reply:
Correct, it is the negation found in the warnings handler. https://github.com/NEstelami/ZAPD/blob/master/ZAPD/WarningHandler.cpp#L132

Reply:
Makes sense. I would add a length check incase `arg < 2`.